CHANGELOG.md

```diff
@@ -22,6 +22,7 @@
 ### Mimirtool
 
 * [BUGFIX] Fix issue where `MIMIR_HTTP_PREFIX` environment variable was ignored and the value from `MIMIR_MIMIR_HTTP_PREFIX` was used instead. #10207
+* [CHANGE] Unify mimirtool authentication options and add extra-headers support for commands that depend on MimirClient. #10178
```

Comment:
```suggestion
* [ENHANCEMENT] Unify mimirtool authentication options and add extra-headers support for commands that depend on MimirClient. #10178
```

As far as I understand, all of this is backwards compatible, right? So there's nothing changing for the user.

Reply:
Yes, that was my intention when doing this PR. Existing workflows should be unaffected by this change and whenever someone would like to use those flags, they would be able to.